client/app/views/partials/pgp/pubkey_input.html

```diff
@@ -1,25 +1,18 @@
-<textarea 
-  pgp-pubkey-validator
-  class="form-control"
-  name="_pubKeyInp"  
-  data-ng-attr-placeholder="{{'Copy and paste the PGP public key here'}}" 
-  data-ng-model="localModel"
-  >
-</textarea>
-<br/>
-<div class="alert alert-danger"
-  data-ng-show="keyForm._pubKeyInp.$error.pgpPubKeyValidator && keyForm._pubKeyInp.$modelValue.$dirty">
-  <p class="has-error"
-  data-translate>
-  The PGP public key you input is invalid!
-  </p>
-</div>
-<div class="alert alert-success"
-  data-ng-show="keyForm._pubKeyInp.$valid && keyForm.$dirty">
-  <p data-translate>
-  The PGP public key you input is valid!
-  </p>
-</div>
-<div data-ng-show="keyForm._pubKeyInp.$valid && keyForm._pubKeyInp.$dirty">
-  <pgp-pubkey-display key-str="localModel">
+<div>
+  <textarea
+    pgp-pubkey-validator
+    class="form-control"
+    name="pgpkey"
+    data-ng-attr-placeholder="{{'Copy and paste the PGP public key here'}}"
+    data-ng-model="localModel"
+    data-ng-required="required"></textarea>
+  <br />
+  <div class="alert alert-danger"
+       data-ng-show="keyForm.pgpkey.$error.pgpPubKeyValidator && keyForm.pgpkey.$modelValue.$invalid">
```

Comment:
better to use $invalid than $dirty.

if i write "a" then i remove "a" that remain marked dirty anyhow.